dev-infrastructure/scripts/kv-copy-cert.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+# Get certificate from a keyvaultS, grant required permissions
+# Grant Secret Officer role to current user, if it is not set
+# Grant Certificate Officer role to current user, if it is not set
```

Comment:
Can we specify which key vault the roles are being set for?

Reply:
It will depend on the parameters that this script is called with